Off-Top-App/off-top-python
# OT-31 | Host MongoDB instance onto GCP

What was done:
Created a MongoDB database using MongoDB Atlas and connected the flask app to the remote database.
How to test this branch:

1. Make sure to run `pip install flask flask_pymongo` if you haven't already
2. Go into the root folder and run `py src\app.py` 
3. Put `http://localhost:5000/sessions` in your browser 
4. If a list of objects with the fields `_id` and `words` appear then everything is working
